src/screens/Appointments/AppointmentBooking.tsx

```diff
@@ -1,25 +1,110 @@
-import React from 'react';
+import React, {useState} from 'react';
 
+import {StyleSheet, Text, View} from 'react-native';
+import DropDownPicker from 'react-native-dropdown-picker';
+import {spacing, theme} from '../../theme';
+import {topics, doctor} from '../../constants/Doctor.constant';
+import {NavigationOptions} from '../../model/Navigation.model';
 import {
-    StyleSheet,
-    Text,
-    View,
-} from 'react-native';
+  appointmentOptionsDate,
+  appointmentOptionsTime,
+} from '../../constants/Appointment.constant';
+import ButtonMain from '../../components/button/ButtonMain';
+import AppointmentOption from '../../components/appointmentItem/AppointmentOption';
 
-const AppointmentBooking = (): JSX.Element => {
+const AppointmentBooking = ({navigation}: NavigationOptions): JSX.Element => {
+  const [open, setOpen] = useState(false);
+  const [value, setValue] = useState(topics[1].topicName);
+  const items = topics.map(topic => ({
+    label: topic.topicName,
+    value: topic.topicName,
+  }));
+  const [openDoctor, setOpenDoctor] = useState(false);
+  const [valueDoctor, setValueDoctor] = useState(doctor[1].doctorName);
+  const itemsDoctor = doctor.map(item => ({

```

Comment:
- rename itemDoctor to make sense
- why doctor.map instead doctors ?
- rename item & add type